locales/ar.json

```
@@ -1,6 +1,143 @@
+    "deleteConfirmation": "هل أنت متأكد من حذف هذا المستخدم؟",
+    "userTypes": {
+      "visitor": "زائر",
+      "speaker": "متحدث",
+      "admin": "مدير"
+    }
+  },
+  "EventManagement": {
+    "title": "إدارة الفعاليات",
+    "addNewEvent": "إضافة فعالية جديدة",
+    "eventsCount": "فعالية",
+    "noDescription": "بدون وصف",
```

Biome (2.1.2)

[error] The key editUser was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)